larpandoracontent/LArVertex/CandidateVertexCreationAlgorithm.cc

```diff
@@ -109,12 +112,37 @@ void CandidateVertexCreationAlgorithm::SelectClusters(ClusterVector &clusterVect
         ClusterVector sortedClusters(pClusterList->begin(), pClusterList->end());
         std::sort(sortedClusters.begin(), sortedClusters.end(), LArClusterHelper::SortByNHits);
 
+        int count1(0);
+        if (m_reducedCandidates)
+        {
+            for (const Cluster *const pCluster : sortedClusters)
+            {
+                float factor1=1;
+
+                if (pCluster->GetParticleId()==E_MINUS)
```

Comment:
Pandora style is to have one space either side of operators such as `==`, `*`, `=`, `<`, `>`, etc.